src/js/config/Categories.js

```diff
@@ -103,6 +112,15 @@ var Categories = [
             "Hex to PEM",
             "Hex to Object Identifier",
             "Object Identifier to Hex",
+            "PGP Encrypt",
+            "PGP Decrypt",
+            "PGP Sign",
+            "PGP Verify",
+            "PGP Sign Cleartext",
+            "PGP Verify Cleartext",
+            "PGP Generate Key Pair",
+            "PGP Add ASCII Armor",
+            "PGP Remove ASCII Armor",
```

Comment:
Change these to 'Sign PGP Cleartext', 'Verify PGP Cleartext', 'Generate PGP Key Pair', 'Add PGP ASCII Armour' and 'Remove PGP ASCII Armour'.